Sources/Client/API/GitHub/GitHubComparison.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+struct GitHubComparison: Decodable {
+  var status: String
```

Comment:
```suggestion
  struct Status: String, Decodable {
    case added
    case removed
    case modified
    case renamed
    case copied
    case changed
    case unchanged
  }
  
  var status: Status
```

Reply:
These don't look like the possible cases of status, are they just placeholders for this example?

Reply:
Sorry I was in a rush, I must've pulled the wrong list from the GitHub documentation.

Reply:
Just noting this here for when I get back 
```
    "status": {
      "type": "string",
      "enum": [
        "diverged",
        "ahead",
        "behind",
        "identical"
      ],
      "examples": [
        "ahead"
      ]
    },
```

Reply:
> Sorry I was in a rush, I must've pulled the wrong list from the GitHub documentation.

You also messed up a little more, because it needs to be an enum (not a struct) and String is not a protocol